charts/wildfly/values.yaml

```diff
@@ -23,6 +23,11 @@ deploy:
       enabled: true
       termination: edge
       insecureEdgeTerminationPolicy: Redirect
+  ingress:
+    enabled: false
+    class: nginx
```

Comment:
This should not be the default as we have no guarantee that nginx controller will be available on the target platform.

Reply:
So i should not provide a default value in Values then ?